packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -32,14 +45,31 @@ export type WebhookDataType = CalendarEvent &
     paymentId?: number;
   };
 
+function addUTCOffset(
+  data: Omit<WebhookDataType, "createdAt" | "triggerEvent">
+): WithUTCOffsetType<WebhookDataType> {
+  if (data.organizer?.timeZone) {
+    (data.organizer as Person & UTCOffset).utcOffset = getUTCOffsetByTimezone(data.organizer.timeZone);
```

Comment:
we are missing the `date` parameter here (`data.startTime`), otherwise, we will always get the utc offset from the current date (important for DST)